templates/thank-you.html

```diff
@@ -16,20 +15,18 @@ <h1 data-qa="title">{{ _("Submission successful") }}</h1>
     "classes": "u-mb-s"
     })
   %}
+    <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
+      survey_title = survey_title)}}</h1>
     <p data-qa="guidance">
-      {% if content.trad_as %}
-        {{ _("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
-          ru_name = content.ru_name,
-          trading_as_name = content.trad_as,
-          submitted_date_time=content.submitted_at|format_datetime) }}
-      {% else %}
-        {{ _("Your answers were submitted for <span>{ru_name}</span> on {submitted_date_time}").format(
-          ru_name = content.ru_name,
-          submitted_date_time=content.submitted_at|format_datetime) }}
-      {% endif %}
+      {{ content.submission_text }}
     </p>
-
-    <p>{{ _("Transaction ID: <b>{transaction_id}</b>").format(transaction_id=content.tx_id)}}</p>
+      {{ onsMetadata({
+        "termCol": 5,
+        "data-qa": "metadata",
+        "descriptionCol": 7,
+        "itemsList": content.metadata_items,
+        })
+      }}
```

Comment:
The spacing in the rendered page looks different to the design:

- The padding all around the green panel is too big - removing `spacious` from the call to `onsPanel` in the template seems to fix this
- The padding at the bottom is still too big, this is because the `onsMetadata` macro is using the `u-mb-l` class - https://ons-design-system.netlify.app/components/metadata/. As this is not configurable it will need a design system change to fix.
- The padding between the green panel and before the text that follows it is too small - changing the `onsPanel` classes from `u-mb-s` to `u-mb-m` seems to fix this
- The submitted on value is missing the word "at" between the date and time
- The submission reference isn't uppercased and the spacing doesn't match the design

![image](https://user-images.githubusercontent.com/1754822/129729014-2c2e5737-e465-4e86-ba97-486dafea6814.png)

vs

![image](https://user-images.githubusercontent.com/1754822/129728879-f020508c-3d1a-4e31-aad6-d7d51b0c0964.png)

Reply:
Also, the spacing between the submitted on and the value for social surveys looks too big:
![image](https://user-images.githubusercontent.com/1754822/129732636-1b026ca8-7b84-4340-854a-21a4605f6090.png)

Reply:
All this is fixed now except hardcoded `u-mb-l` class problem in onsMetadata which, as you said, needs some DS fix. Columns of metadata have different values now depending on survey type but I got them as close to the prototype as I could without DS fix. Also, metadata values moved to `submission_metadata_context` file as we discussed. Thanks!